sdk/spring/spring-cloud-azure-service/src/main/java/com/azure/spring/cloud/service/implementation/redis/AzureJedisClientConfig.java

```diff
@@ -0,0 +1,230 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.spring.cloud.service.implementation.redis;
+
+import redis.clients.jedis.HostAndPortMapper;
+import redis.clients.jedis.JedisClientConfig;
+import redis.clients.jedis.Protocol;
+
+import javax.net.ssl.HostnameVerifier;
+import javax.net.ssl.SSLParameters;
+import javax.net.ssl.SSLSocketFactory;
+import java.util.Objects;
+import java.util.function.Supplier;
+
+/**
+ * JedisClientConfig implementation used for Azure Redis.
+ */
+public final class AzureJedisClientConfig implements JedisClientConfig {
```

Comment:
Should we put these in the autoconfigure module too?

Reply:
Follow the pattern: jedis, spring data, auto-configure:
I think the implmentation belongs to jedis level.